sample/SampleRobot/samplerobot_carry_object.py

```diff
@@ -91,7 +91,7 @@ def init ():
     hcf.ic_svc.setImpedanceControllerParam("larm", icp)
     hcf.Groups = defJointGroups()
     hcf.startDefaultUnstableControllers(['rarm', 'larm'], ["rleg", "lleg", "rarm", "larm"])
-    HRPSYS_DIR=check_output(['pkg-config', 'hrpsys-base', '--variable=prefix']).rstrip()
+    HRPSYS_DIR=check_output(['pkg-config', 'hrpsys-base', '--variable=prefix']).rstrip().decode()
```

Comment:
may be need to check if this works with 18.04 environment.  @takuma-hiraoka Could you check this?

Reply:
In my melodic environment, launching samplerobot simulation with `TORQUR_CONTROL:=true` failed. Robot hands dropped down immediately and the samplerobot fell down backward, so I could not test it in melodic.

Reply:
OK